[2022-02-15 02:29:17,074] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.dim_customers manual__2022-02-15T02:27:57.149490+00:00 [queued]>
[2022-02-15 02:29:17,098] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.dim_customers manual__2022-02-15T02:27:57.149490+00:00 [queued]>
[2022-02-15 02:29:17,102] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 02:29:17,105] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 02:29:17,106] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 02:29:17,129] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): model.my_new_project.dim_customers> on 2022-02-15 02:27:57.149490+00:00
[2022-02-15 02:29:17,137] {standard_task_runner.py:52} INFO - Started process 926 to run task
[2022-02-15 02:29:17,158] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'model.my_new_project.dim_customers', 'manual__2022-02-15T02:27:57.149490+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmp798ikg04', '--error-file', '/tmp/tmpjekje1gp']
[2022-02-15 02:29:17,161] {standard_task_runner.py:77} INFO - Job 77: Subtask model.my_new_project.dim_customers
[2022-02-15 02:29:17,303] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.model.my_new_project.dim_customers manual__2022-02-15T02:27:57.149490+00:00 [running]> on host de7a240d0d46
[2022-02-15 02:29:17,407] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=model.my_new_project.dim_customers
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T02:27:57.149490+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T02:27:57.149490+00:00
[2022-02-15 02:29:17,410] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 02:29:17,413] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json run --target dev --models dim_customers             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 02:29:17,454] {subprocess.py:85} INFO - Output:
[2022-02-15 02:29:24,465] {subprocess.py:89} INFO - 02:29:24  Running with dbt=1.0.1
[2022-02-15 02:29:25,199] {subprocess.py:89} INFO - 02:29:25  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2022-02-15 02:29:25,200] {subprocess.py:89} INFO - There are 2 unused configuration paths:
[2022-02-15 02:29:25,201] {subprocess.py:89} INFO - - models.jaffle_shop.marts.core
[2022-02-15 02:29:25,203] {subprocess.py:89} INFO - - models.jaffle_shop.staging
[2022-02-15 02:29:25,205] {subprocess.py:89} INFO - 
[2022-02-15 02:29:25,261] {subprocess.py:89} INFO - 02:29:25  Found 7 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 02:29:25,268] {subprocess.py:89} INFO - 02:29:25
[2022-02-15 02:29:31,096] {subprocess.py:89} INFO - 02:29:31  Concurrency: 4 threads (target='dev')
[2022-02-15 02:29:31,102] {subprocess.py:89} INFO - 02:29:31
[2022-02-15 02:29:31,114] {subprocess.py:89} INFO - 02:29:31  1 of 1 START view model dbt_bytecode_training.dim_customers..................... [RUN]
[2022-02-15 02:29:33,586] {subprocess.py:89} INFO - 02:29:33  1 of 1 OK created view model dbt_bytecode_training.dim_customers................ [SUCCESS 1 in 2.47s]
[2022-02-15 02:29:33,641] {subprocess.py:89} INFO - 02:29:33
[2022-02-15 02:29:33,644] {subprocess.py:89} INFO - 02:29:33  Finished running 1 view model in 8.37s.
[2022-02-15 02:29:33,651] {subprocess.py:89} INFO - 02:29:33
[2022-02-15 02:29:33,653] {subprocess.py:89} INFO - 02:29:33  Completed successfully
[2022-02-15 02:29:33,655] {subprocess.py:89} INFO - 02:29:33
[2022-02-15 02:29:33,658] {subprocess.py:89} INFO - 02:29:33  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-02-15 02:29:34,164] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 02:29:34,200] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=model.my_new_project.dim_customers, execution_date=20220215T022757, start_date=20220215T022917, end_date=20220215T022934
[2022-02-15 02:29:34,241] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 02:29:34,315] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
